docs/en/sql-reference/functions/array-functions.md

```diff
@@ -1794,6 +1794,55 @@ Return value type is always [Float64](../../sql-reference/data-types/float.md).
 │ 6   │ Float64                                                                                  │
 └─────┴──────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+## arrayRemove
```

Comment:
It does not describe the usual way to call the function:

arrayRemove(element, array)